app/controllers/tweets_controller.rb

```diff
@@ -0,0 +1,105 @@
+class TweetsController < ApplicationController
+    
+  get '/tweets/new' do
+    if logged_in
+      erb :'/tweets/new'
+    else
+      redirect '/login'
+    end
+  end
+
+  post '/tweets' do
+    if logged_in
+      @user = current_user
```

Comment:
current_user is already saved as an instance variable from the parent class that we are inheriting from, so this just sets two instance variables.